style.css

```diff
@@ -0,0 +1,81 @@
+ul.header {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
```

Comment:
- [ ] Kindly please make sure you don't write default value here. So you can make your code clean. 🆗

Reply:
- [ ] Remember that the comment should be a suggestion instead of making it look like an order.

Reply:
- [ ] I think you have to remove those two lines because they are default values. 🆗 
```
    margin: 0;
    padding: 0;
```
Such like coding practice will cause `CSS` linter errors